DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```
@@ -360,32 +414,91 @@ private void RegisterCommands(IEnumerable<Type> types, ulong? guildid)
                 {
                     try
                     {
+                        async Task UpdateCommandPermission(ulong commandId, string commandName, Type commandDeclaringType, Type commandRootType)
+                        {
+                            if (guildid == null)
+                            {
+                                // Not supported by V9 ATM
```

Perhaps throw a `NotImplementedException` instead of silently skipping it?